source/v0/TCPStream.cpp

```diff
@@ -68,3 +68,14 @@ void TCPStream::_eventHandler(struct socket_event *ev)
             break;
     }
 }
+
+void TCPStream::setNagle(bool enable)
+{
+    void * enable_ptr;
+    if(enable) {
+        enable_ptr = (void*)1;
```

Comment:
@bremoran This is bit strange. Why are you using the ptr to pass value, rather than the value of the ptr?